exasol_integration_test_docker_environment/main.py

```diff
@@ -1,12 +1,16 @@
 #! /usr/bin/env python3
 #
 from exasol_integration_test_docker_environment.cli.cli import cli
-import exasol_integration_test_docker_environment.cli.commands
 
 
 def main():
+    # required so the cli will print the available subcommands
+    from exasol_integration_test_docker_environment.cli.commands import (
```

Comment:
There is I think another option to achieve this, by adding this import statement to the __init__.py for commands package